docs/manual/java/guide/services/ServiceClients.md

```diff
@@ -59,11 +59,13 @@ All service calls with Lagom service clients are by default using circuit breake
 
 In the above example the default identifier is used for the `sayHi` method, since no specific identifier is given. The default identifier is the same as the service name, i.e. `"hello"` in this example. The `hiAgain` method will use another circuit breaker instance, since `"hello2"` is specified as circuit breaker identifier.
 
+### Circuit Breaker Configuration
+
 On the client side you can configure the circuit breakers. The default configuration is:
 
 @[circuit-breaker-default](../../../../../service/core/client/src/main/resources/reference.conf)
 
-That configuration will be used if you don't define any configuration yourself.
+That configuration will be used if you don't define any configuration yourself. Lagom's client [[maps all 4xx and 5xx responses to Exceptions|ServiceErrorHandling]] and Lagom's Circuit Breaker defaults to considering all Exceptions as failures. Sometimes you want to configure the circuit breaker for a given endpoint so it ignores a certain exception. This is particularly useful when connecting to services where 4xx HTTP status codes are used to model business valid cases. For example, it may be a non-failure case to respond a 404 Not Found to a particular request. In that case the client will still receive a `NotFound` exception as a result of invoking the service but you can add `"com.lightbend.lagom.scaladsl.api.transport.NotFound"` to the circuit breaker whitelist so that it is not considered by the circuit breaker failure count.
```

Comment:
I'm still finding it really difficult to parse this section because of the way it is organized. To me, it makes it much simpler to separate the two thoughts, the default behavior and then why and how to configure it, something like this:

By default, Lagom's Circuit Breaker [[maps all 4xx and 5xx responses to Exceptions|ServiceErrorHandling]] and considers all Exceptions as failures. This behavior is defined in the default configuration:

@[circuit-breaker-default](../../../../../service/core/client/src/main/resources/reference.conf)

You can change the default behavior by whitelisting particular exceptions so they do not count as failures. In some cases, 4xx HTTP status codes model business valid cases. For example, it may be a non-failure case to respond with a 404 Not Found code. In this case, add "com.lightbend.lagom.scaladsl.api.transport.NotFound"` to the circuit breaker whitelist. The client will still receive a `NotFound` exception as a result of invoking the service, but it will not be considered a failure by the Circuit Breaker.

Reply:
Thanks @rstento, I added some extra introduction just after the [default] snippet. The thing is that the snippet itself already includes documentation so the final page could be a little toto verbose. Still, I think this intro is helpful. I've made some other small fixes with you suggestions in mind.